scene/2d/canvas_item.cpp

```diff
@@ -1118,6 +1118,17 @@ void CanvasItem::force_update_transform() {
 	notification(NOTIFICATION_TRANSFORM_CHANGED);
 }
 
+void CanvasItem::set_culling_rect(const Rect2 &p_rect) {
+
+	culling_rect = p_rect;
+
+	VS::get_singleton()->canvas_item_set_custom_rect(get_canvas_item(), false, p_rect);
+}
```

Comment:
```suggestion
void CanvasItem::set_visibility_rect(const Rect2 &p_rect) {

	visibility_rect = p_rect;

	VS::get_singleton()->canvas_item_set_custom_rect(get_canvas_item(), false, p_rect);
}
```

The `VisualServer` API is not in alignment with either name, but naming this as `visibility_rect` allows to group the property in the inspector under `Visibility`. But looking at `Particles2D`, it has the same property already... Which in fact could be reused perhaps, but not suitable for `3.2`.